app/lib/irs1040_schedule_eic_pdf.rb

```diff
@@ -0,0 +1,43 @@
+class Irs1040ScheduleEicPdf
+  include PdfHelper
+
+  def source_pdf_name
+    "tax_documents/f1040sei-TY2021"
+  end
+
+  def initialize(submission)
+    @xml_document = SubmissionBuilder::Ty2021::Documents::ScheduleEic.new(submission).document
+    @intake = submission.intake
+  end
+
+  def hash_for_pdf
+    answers = {
+      FullPrimaryName: @intake.primary_full_name,
+      PrimarySSN: @intake.primary_ssn
+    }
+    dependent_nodes = @xml_document.search("QualifyingChildInformation")
+    answers.merge!(dependents_info(dependent_nodes[0..]))
+    answers
```

Comment:
Since the return value of merge! is the entire new hash, I think that returning answers is redundant. But also, it is more clear...

![Screen Shot 2022-07-28 at 11 55 12 AM](https://user-images.githubusercontent.com/4494389/181616104-4d67617b-cd45-4c08-929a-04eadc03fb29.png)